worker/uniter/runner/debug/server.go

```diff
@@ -102,8 +102,14 @@ export | grep -v $FILTER > $JUJU_DEBUG/env.sh
 cat > $JUJU_DEBUG/welcome.msg <<END
 This is a Juju debug-hooks tmux session. Remember:
 1. You need to execute hooks manually if you want them to run for trapped events.
-2. When you are finished with an event, you can run 'exit' to close the current window and allow Juju to continue running.
-3. CTRL+a is tmux prefix.
+2. When you are finished with an event, you can run 'exit' to close the current window and allow Juju to continue processing
+new events for this unit without exiting a current debug-session.
+3. To run a hook, end a debugging session and avoid processing any more events manually use
```

Comment:
For the case where you have multiple events queued-up but you have ran juju debug-hooks unconditionally:

juju debug-hooks # <nothing_here>

Sometimes you might want to end a session without executing queued hooks manually and just let a unit agent do it without further interruptions.